hmftools.wdl

```diff
@@ -0,0 +1,927 @@
+version 1.0
+
+# Copyright (c) 2020 Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+task Amber {
+    input {
+        String normalName
+        File normalBam
+        File normalBamIndex
+        String tumorName
+        File tumorBam
+        File tumorBamIndex
+        String outputDir = "./amber"
+        File loci
+        File referenceFasta
+        File referenceFastaFai
+        File referenceFastaDict
+
+        Int threads = 2
+        String memory = "70G"
+        String javaXmx = "64G"
+        Int timeMinutes = 240
+        String dockerImage = "quay.io/biocontainers/hmftools-amber:3.5--0"
+    }
+
+    command {
+        AMBER -Xmx~{javaXmx} \
+        -reference ~{normalName} \
+        -reference_bam ~{normalBam} \
+        -tumor ~{tumorName} \
+        -tumor_bam ~{tumorBam} \
+        -output_dir ~{outputDir} \
+        -threads ~{threads} \
+        -ref_genome ~{referenceFasta} \
+        -loci ~{loci}
+    }
+
+    output {
+        File version = "~{outputDir}/amber.version"
+        File tumorBafPcf = "~{outputDir}/~{tumorName}.amber.baf.pcf"
+        File tumorBafTsv = "~{outputDir}/~{tumorName}.amber.baf.tsv"
+        File tumorBafVcf = "~{outputDir}/~{tumorName}.amber.baf.vcf.gz"
+        File tumorBafVcfIndex = "~{outputDir}/~{tumorName}.amber.baf.vcf.gz.tbi"
+        File tumorContaminationVcf = "~{outputDir}/~{tumorName}.amber.contamination.vcf.gz"
```

Comment:
Having the name "Amber" right next to contamination and tumorName is quite distracting. I know people with that name. Anyway I don't see how we can improve on the naming scheme.